### ISSUE TITLE
Website

### ISSUE THREAD

**Issue by reveurguy** [kubewarden/kubewarden.io#158]
Hi, I am new to Kubewarden and want to contribute to improve the website design , look(style wise) as well as work on the Performance, SEO, Accessibility of the website.
As this snapshot displays, the performance, seo and accessibility can be improved upon.

<img width="470" alt="Screenshot 2022-12-12 at 10 23 28 PM" src="https://user-images.githubusercontent.com/59417546/207105446-00a27f0c-6272-434a-a880-4a89c779728b.png">
This was taken with lighthouse.

**Comment by flavio** [comment]
Hi @reveurguy, thanks for reporting this issue and being interested to help the kubewarden project.

We are not experts on this topics, any kind of help is apprecitated! :hugs:

**Comment by reveurguy** [comment]
Yeah, I'll make some changes and send a PR then we can build upon that

**Comment by reveurguy** [comment]
@flavio I have made some initial changes to design of the site. Please provide your feedback for that
